src/Normalizer/Normalizer.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Http\Client\Plugin\Normalizer;
+
+use Psr\Http\Message\RequestInterface;
+use Psr\Http\Message\ResponseInterface;
+
+/**
+ * Normalize a request or a response into a string or an array
```

Comment:
Hum i add `@internal` on the method, but this normalizer is injected into the plugin, to allow user to change how to render the log line, so it's not that "internal" ?